docs/_data/en/toc_text.yml

```diff
@@ -3,6 +3,7 @@
 # For documentation, see https://developer.shotgunsoftware.com/tk-doc-generator/authoring/toc/
 #
 
+
```

Comment:
did you mean to put an extra line in here?

Reply:
i didn't, and it's gone